README.md

```diff
@@ -1,14 +1,6 @@
 wikidata-analysis
 =================
-see [tools.wmflabs.org/wikidata-analysis](http://tools.wmflabs.org/wikidata-analysis)
 
-The script is run daily on wmflabs using the following cron job
->0 4 * * * jsub -once -mem 6G -N analysis /data/project/wikidata-analysis/src/run.labs.wikidata-analysis.sh.sh
+Written in Java using the Wikidata-Toolkit.
 
-
-#### to run in other environments
-To run in other environments just run run.sh
-
-* NOTE: the scripts require a lot of memory
-* NOTE: the scripts expect public_html to be in your home dir
-* NOTE: analysis.sh expects to be run when in its path
\ No newline at end of file
+Please run your Java VM with -Xmx2g or greater.
```

Comment:
This readme is out of date / in the wrong place:
See https://github.com/wmde/wikidata-analysis/blob/addshoreRewrite/java/analyzer/README.md
I have removed the line above (in the next commit)